View components & environment flow while viewing details

jeff-phillips-18 opened the pull request:
## Fixes 
Fixes https://issues.redhat.com/browse/HAC-2151

## Description
Updates the application details view to use the environment cards as tabs to show the application environment details view below on the same page.

Updates PF versions to pick fix for topology view sizing

## Type of change

- [x] Feature

## Screen shots / Gifs for design review 
![image](https://user-images.githubusercontent.com/11633780/193277600-160ffa01-da52-4c46-8ab0-4ab97104290c.png)

![image](https://user-images.githubusercontent.com/11633780/193277670-79ece13c-2ef9-437e-bbb0-d71958ee2430.png)

![image](https://user-images.githubusercontent.com/11633780/193277716-69297048-9867-436b-aa7a-acd540eced0f.png)

![image](https://user-images.githubusercontent.com/11633780/193277764-a7d6e900-f148-4381-b765-a9af3b9f8ecf.png)

![image](https://user-images.githubusercontent.com/11633780/193277825-a9559a26-5cfe-4cd8-ab14-2d396a7b4cd0.png)

/cc @itsptk @serenamarie125 @christianvogt @karthikjeeyar

karthikjeeyar commented on src/components/Environment/ApplicationEnvironmentCards.tsx:
```diff
@@ -74,58 +89,43 @@ const ApplicationEnvironmentCard: React.FC<ApplicationEnvironmentCardProps> = ({
+                environment={env}
+                isExpanded={cardsExpanded}
+                isSelected={env.metadata.name === selectedEnvironment}
+                onSelect={() => onSelect && onSelect(env.metadata.name)}
```
we can use same displayName in the heading below like we have in card title.

![image](https://user-images.githubusercontent.com/9964343/193847990-bab6552d-5dc8-4fc8-9538-5e7825fe37a1.png)


```suggestion
                onSelect={() => onSelect && onSelect(env.spec.displayName)}
```

rohitkrai03 replied:
Agreed, I think it would be better to use display name here as well.

karthikjeeyar commented:
When there is getting started card on the top of the page, then the graph area is too small to work with (height 150px). 
 
![image](https://user-images.githubusercontent.com/9964343/193849629-0015ac95-58fb-48d3-a98f-1a4ff7955b6d.png)

rohitkrai03 commented:
/retest

rohitkrai03 reviewed (approved):
/approve

rohitkrai03 commented:
@jeff-phillips-18 Can you update the PR with the small change that @karthikjeeyar suggested? We can merge the PR after that.

jeff-phillips-18 commented:
@rohitkrai03 Updated and squashed. Thanks!

karthikjeeyar commented:
/lgtm

jeff-phillips-18 commented:
@karthikjeeyar I've reverted your requested change and updated only where we display the title to use the `spec.displayName`. Using the displayName for determining selection could be problematic since it is not guaranteed to be unique.

karthikjeeyar commented:
/lgtm
